hooks/post_gen_project.py

```diff
@@ -1,101 +1,167 @@
+from json import load
 from pathlib import Path
 from shutil import rmtree
-from typing import Union
+from typing import List, Union
 
 
-def remove_folder(folder: Union[Path, str]) -> None:
-    """Remove a folder.
+def delete_files_and_folders(paths: Union[Path, str, List[Path], List[str]]) -> None:
+    """Delete files and folders for given file and/or folder paths.
 
     Args:
-        folder: A folder path to be removed.
+        paths: A ``pathlib.Path`` object or string, or list of ``pathlib.Path`` or
+        strings.
 
     Returns:
-        None - removes the `folder` folder.
+        None - deletes the files and/or folders defined in ``paths``.
 
```

Comment:
Worth mentioning what happens in case there's an error (rmtree failed, etc)?

Reply:
I've added more information in the docstring, but will also test what happens with Windows users who may not have admin privileges (GitHub Actions Windows build by default have these privileges). [Concerns based on this SO post](https://stackoverflow.com/questions/2656322/shutil-rmtree-fails-on-windows-with-access-is-denied). Will report back here (although may raise an issue rather than fix it here).

Reply:
To confirm, with a Standard Windows account, I can run `govcookiecutter`, and this function does not raise any errors, and the files are deleted as expected. Note I ran this within the `Documents` folder, so [creating a new project in a folder that requires user access control](https://docs.microsoft.com/en-us/windows/security/identity-protection/user-account-control/how-user-account-control-works) may fail.

Reply:
ok